### ISSUE TITLE
combo_type a NaN/inf

### ISSUE THREAD

**Issue by jsitarek** [cta-observatory/magic-cta-pipe#224]
I got an error while processing one of LST subruns: dl1_LST-1.Run15757.0043.h5

```
Average offset: 118.780 us
--> Number of coincident events: 1019
--> Fraction over the MAGIC-II events: 62.7%

In total 1566 stereo events are found:
Traceback (most recent call last):
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/bin/lst1_magic_event_coincidence", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/fefs/home/julian.sitarek/prog/mcp_ctapipe019/magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py", line 635, in main
    event_coincidence(
  File "/fefs/home/julian.sitarek/prog/mcp_ctapipe019/magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py", line 535, in event_coincidence
    event_data = get_stereo_events(event_data, config)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/fefs/home/julian.sitarek/prog/mcp_ctapipe019/magicctapipe/io/io.py", line 415, in get_stereo_events
    event_data_stereo = event_data_stereo.astype({"combo_type": int})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/generic.py", line 6617, in astype
    res_col = col.astype(dtype=cdt, copy=copy, errors=errors)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/generic.py", line 6640, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/internals/managers.py", line 430, in astype
    return self.apply(
           ^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/internals/managers.py", line 363, in apply
    applied = getattr(b, f)(**kwargs)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/internals/blocks.py", line 758, in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/dtypes/astype.py", line 237, in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/dtypes/astype.py", line 182, in astype_array
    values = _astype_nansafe(values, dtype, copy=copy)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/dtypes/astype.py", line 101, in _astype_nansafe
    return _astype_float_to_int_nansafe(arr, dtype, copy)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/julian.sitarek/anaconda3/envs/magic-lst_0_19/lib/python3.11/site-packages/pandas/core/dtypes/astype.py", line 145, in _astype_float_to_int_nansafe
    raise IntCastingNaNError(
pandas.errors.IntCastingNaNError: Cannot convert non-finite values (NA or inf) to integer: Error while type casting for column 'combo_type'

```

**Comment by jsitarek** [comment]
fixed by #226

**Comment by jsitarek** [comment]
fixed by #226
